db/migrate/20210125051022_devise_create_users.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+class DeviseCreateUsers < ActiveRecord::Migration[6.1]
+  def change
+    create_table :users do |t|
+      ## Database authenticatable
+      t.string :email,              null: false, default: ''
+      t.string :encrypted_password, null: false, default: ''
+      t.string :username,           null: false, default: ''
+      t.integer :rating,            null: true
+      t.integer :resources,         null: true
```

Comment:
Should resources and rating be nullable? Unsure if this is due to a devise limitation or not

Reply:
no, they shouldn't, and no, it's not a devise limitation

Reply:
Resources and rating no longer null and default to 0 for now, likely to change as starting player rating/resources are further discussed 👍